LICENSE

```diff
@@ -1,6 +1,7 @@
 BESS is distributed under the following BSD3 license:
 
-Copyright (c) 2014-2016, Regents of the University of California and Nefeli Networks, Inc.  All rights reserved.
+Copyright (c) 2014-2016, Regents of the University of California.  All rights reserved.
```

Comment:
Will update.  My habit has always been two spaces after periods, but I know that's less common these days:

https://en.wikipedia.org/wiki/History_of_sentence_spacing#French_and_English_spacing